manifests/init.pp

```diff
@@ -0,0 +1,176 @@
+  Integer[0,1]                $auto_x_detect                     = 0,
+  Optional[Array[String[1]]]  $bindir                            = undef,
+  Integer[0,1]                $color_set2                        = 0,
+  Integer[0,1]                $copy_log_on_error                 = 0,
+  String[1]                   $dbdir                             = '/var/lib/rkhunter/db',
+  Optional[Array[String[1]]]  $disable_tests                     = undef,
+  Optional[Array[String[1]]]  $empty_logfiles                    = undef,
+  Array[String[1]]  $enable_tests                                = ['ALL'],
+  Optional[String[1]]         $epoch_date_cmd                    = undef,
+  Optional[Array[String[1]]]  $exclude_user_fileprop_files_dirs  = undef,
+  Optional[Array[String[1]]]  $existwhitelist                    = undef,
+  Integer[0,1]                $globstar                          = 0,
+  String[1]                   $hash_cmd                          = 'SHA256',
+  Integer[1]                  $hash_fld_idx                      = 1,
+  Optional[Array[String[1]]]  $ignore_prelink_dep_err            = undef,
+  Integer[0,1]                $immutable_set                     = 0,
+  Optional[Array[String[1]]]  $immutwhitelist                    = undef,
+  Optional[Array[String[1]]]  $inetd_allowed_svc                 = undef,
+  Optional[String[1]]         $inetd_conf_path                   = undef,
+  String[1]                   $installdir                        = '/usr',
+  Integer[1]                  $ipc_seg_size                      = 1048576,
+  String[1]                   $language                          = 'en',
+  Integer[1]                  $lock_timeout                      = 300,
+  String[1]                   $lockdir                           = '/var/run/lock',
+  String[1]                   $logfile                           = '/var/log/rkhunter/rkhunter.log',
+  Optional[String[1]]         $mail_on_warning                   = undef,
+  String[1]                   $mail_cmd                          = 'mail -s "[rkhunter] Warnings found for ${HOST_NAME}"', # lint:ignore:single_quote_string_with_variables
+  Integer[0,2]                $mirrors_mode                      = 0,
+  Optional[Array[String[1]]]  $missing_logfiles                  = undef,
+  Optional[String[1]]         $modules_dir                       = undef,
+  Optional[String[1]]         $os_version_file                   = undef,
+  String[1]                   $password_file                     = '/etc/shadow',
+  Integer[0,1]                $phalanx2_dirtest                  = 0,
+  Optional[Array[String[1]]]  $pkgmgr_no_vrfy                    = undef,
+  String[1]                   $pkgmgr                            = 'RPM',
+  Optional[Array[String[1]]]  $port_path_whitelist               = undef,
+  Optional[Array[String[1]]]  $port_whitelist                    = undef,
+  Optional[Array[String[1]]]  $pwdless_accounts                  = undef,
```

Comment:
Does a SIMP server or a SIMP-managed workstation have any password-less-accounts that need to be listed here?

Reply:
I didn't think any password-less accounts would be allowed with our security guidelines. I would have to dig into that though as I am not sure what all accounts are made.

Reply:
I'm actually not quite sure what this is checking for. I ran it on a few SIMP systems in different configurations and it didn't die so keeping it unset seems to be fine.